backend/eventloop/shared_resource_loop/sharedresourceloop.go

```diff
@@ -530,7 +530,7 @@ func internalProcessMessage_GetOrCreateClusterUserByNamespaceUID(ctx context.Con
 	if err != nil {
 		if db.IsResultNotFoundError(err) {
 			isNewUser = true
-
+			clusterUser.Display_name = string(workspaceNamespace.Name)
```

Comment:
`workspaceNamespace.Name` is already of type string. I don't think we need to explicitly convert it again.